README.md

```diff
@@ -77,110 +77,87 @@ This accepts an array of the loggers' codes which will be used to select the log
 
 **1. Base Logger**
 
-The base logger only with the following format using a `message`, `action` and all remaining arguments are rendered in a JSON payload
+The base logger only with the following format using a `message`,
+`action` and all remaining arguments are rendered in a JSON payload
 
 ```ruby
 # Sample usage
 EventTracer.info action: 'Action', message: 'Message', other_args: 'data'
 => "[Action] message {\"other_args\":\"data\"}"
 ```
 
-**2. Appsignal**
+**2. Metrics**
 
-Appsignal 2.5.1 is currently supported for the following metric functions available for the EventTracer's log methods
+EventTracer allows sending metrics together with your log. Currently two
+monitoring services are supported: AppSignal and DataDog.
 
-- increment_counter
-- add_distribution_value
-- set_gauge
+All metrics are sent in `metrics` fields, for example:
 
-All other functions are exposed transparently to the underlying Appsignal class
-
-The interface for using the Appsignal wrapper is:
+```ruby
+EventTracer.info(
+  action: 'Action',
+  message: 'There is an action',
+  metrics: [
+    :metric_1,
+    { metric_2: :gauce },
```

Comment:
```suggestion
    { metric_2: :gauge },
```

defaults to 1? 🤔 since gauge is usually a non-counter value like latency to be avg-ed/ percentile assessed later

Reply:
Actually, this wouldn't be the case, I'll remove it. There will be only 2 syntax:

1. Array

```ruby
metrics: [:counter_1, :counter_2]
```

2. Hash

```ruby
metrics: {
  gauge_1: { type: :gauge, value: 123 },
  counter_2: { type: :counter, value: 1 }
}
```